integration-test/jest/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@integration-test/jest",
+  "scripts": {
+    "prepare": "rm -rf node_modules/@apollo/client-react-streaming node_modules/@apollo/experimental-nextjs-app-support; mkdir -p node_modules/@apollo/client-react-streaming node_modules/@apollo/experimental-nextjs-app-support && cp -r ../../packages/client-react-streaming/{package.json,dist} node_modules/@apollo/client-react-streaming && cp -r ../../packages/experimental-nextjs-app-support/{package.json,dist} node_modules/@apollo/experimental-nextjs-app-support",
+    "test": "yarn prepare; jest"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.9.1",
```

```suggestion
    "@apollo/client": "^3.9.5",
```

I know it shouldn't matter, but since we are here, should this reflect the latest version?

---

fair (although CI will always install the latest version anyways ^^)